Packs/Netcraft_V2/Integrations/Netcraft/Netcraft.yml

```diff
@@ -0,0 +1,1501 @@
+      type: String
+    - contextPath: Netcraft.Submission.classification_log.date
+      description: A unix timestamp of when this state change occurred. This key appears only if the "submission_uuid" argument is provided.
+      type: Number
+    - contextPath: Netcraft.Submission.classification_log.from_state
+      description: The state the submission moved out of. This key appears only if the "submission_uuid" argument is provided.
+      type: String
+    - contextPath: Netcraft.Submission.classification_log.to_state
+      description: The state the submission moved into. This key appears only if the "submission_uuid" argument is provided.
+      type: String
+    - contextPath: Netcraft.Submission.files
+      description: An API URL to get details about the files associated with this submission. This key appears only if the "submission_uuid" argument is provided.
+      type: String
+    - contextPath: Netcraft.Submission.has_cryptocurrency_addresses
+      description: Whether the submission contains cryptocurrency addresses. This key appears only if the "submission_uuid" argument is provided.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_files
+      description: Whether the submission contains files. This key appears only if the "submission_uuid" argument is provided.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_issues
+      description: Whether the submission contains issues. This key appears only if the "submission_uuid" argument is provided.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_mail
+      description: Whether the submission has a mail. This key appears only if the "submission_uuid" argument is provided.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_phone_numbers
+      description: Whether the submission contains phone numbers. This key appears only if the "submission_uuid" argument is provided.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_urls
+      description: Whether the submission contains URLs. This key appears only if the "submission_uuid" argument is provided.
+      type: Boolean
+    - contextPath: Netcraft.Submission.is_archived
+      description: Whether the submission has been archived. This key appears only if the "submission_uuid" argument is provided.
+      type: Boolean
+    - contextPath: Netcraft.Submission.last_update
+      description: A unix timestamp of when this submission was last updated. This key appears only if the "submission_uuid" argument is provided.
+      type: Number
+    - contextPath: Netcraft.Submission.mail
+      description: An API URL to get details about the mail associated with this submission. This key appears only if the "submission_uuid" argument is provided.
+      type: String
+    - contextPath: Netcraft.Submission.mail_state
+      description: The state of the mail in the submission. This key appears only if the "submission_uuid" argument is provided.
```

Comment:
Should this be email?
- An API URL to get details about the email associated with this submission
- The state of the email in the submission.